client/boot/project/wordpress-com.js

```diff
@@ -5,19 +5,18 @@ import { includes, startsWith } from 'lodash';
 import React from 'react';
 import ReactDom from 'react-dom';
 import store from 'store';
-import debugFactory from 'debug';
-const debug = debugFactory('calypso');
 import page from 'page';
+import debugFactory from 'debug';
 
 /**
  * Internal dependencies
  */
 import config from 'config';
 
-import abtestModule from 'lib/abtest'; // used by error logger
+import { getSavedVariations } from 'lib/abtest'; // used by error logger
```

Comment:
why did this need to be changed?

Reply:
Well, calling `abtestModule.getSavedVariations()` broke -- I guess because `import abtest` would look for a `default` export (and there is none in `lib/abtest`)...?

Reply:
That makes sense. `require` doesn't need a default, but for `import` to work the same way it would need to be written `import name as * from 'source'`.

Do you think it could be a good idea to fix these cases before running the codemod?

Reply:
> Do you think it could be a good idea to fix these cases before running the codemod?

Yes!